.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -248,22 +249,21 @@ steps:
 
       - label: ":linux: Cloud (MODULE) Tests"
         key: "x-pack-metricbeat-extended-cloud-test"
-        skip: "doesn't belong in a stage in Jenkins, thus skipped"
-        if: build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
-        # see link in Jenkins: https://github.com/elastic/beats/blob/ccd7b135df70358f8a02393d9bd8b716428b8048/x-pack/metricbeat/Jenkinsfile.yml#L39
-        # additionally skipping due to https://github.com/elastic/ingest-dev/issues/3170
+        skip: "Not a stage on Jenkins & elastic/ingest-dev#3170"
+        # https://github.com/elastic/ingest-dev/issues/3170
+        if: build.env("BUILDKITE_PULL_REQUEST") == "false"  || build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
```

Comment:
```suggestion
        if: build.env("GITHUB_PR_LABELS") =~ /.*aws.*/
```

Reply:
we don't want to run this AWS Cloud test (based on my understanding of what was done in Jenkins as described in https://github.com/elastic/beats/pull/39533) -- in this step we anyway use `skip:` so it won't run, but it would be good if the conditional is also correct for the future.